### PR TITLE
holoviews 1.15.2

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Jira ticket:** [PKG-725](https://anaconda.atlassian.net/browse/PKG-725) (holoviews-1.15.2)

**The upstream data:**
Github releases:  https://github.com/ioam/holoviews/releases
[Diff between the latest and previous upstream releases](https://github.com/holoviz/holoviews/compare/v1.15.0...v1.15.2)
Changelog: https://github.com/ioam/holoviews/blob/master/CHANGELOG.md
Requirements:
 * setup.py:  https://github.com/ioam/holoviews/blob/v1.15.2/setup.py
 *  pyproject.toml:  https://github.com/ioam/holoviews/blob/v1.15.2/pyproject.toml

**_Actions:_**

1. Remove pinnings from `host`.
2. Add pinning for `param`
3. Add `packaging` to `run`
4. Fix `bokeh` pinning because versions `>= 3.0.0` have breaking changes 

**Package's statistics**
<details>

 * Priority B | effort: easy | Category: anaconda_affiliated | subcategory:  | pkg_type: python
 * Outdated platfroms: 'linux-64', 'linux-aarch64', 'linux-ppc64le', 'osx-64', 'osx-arm64', 'win-64'
 * Latest version: 1.15.2
 * Release on PyPi:
    * version: 1.15.2
    * date: 2022-11-03T13:35:06
* [PyPi history](https://pypi.org/project/holoviews/#history)
 * Popularity: 
    * 3 months downloads: 348828.0
    * All time:  965065 downloads -  holoviews  1.15.2  Stop plotting your data - annotate your data and let it visualize itself.  

</details>

**Other checks:**

<details>

5. - [x] https://holoviews.org is present
6. - [x] Check the pinnings
8. - [x] Additional research
    https://github.com/holoviz/holoviews/issues
9. - [x] `dev_url`:
    https://github.com/holoviz/holoviews
10. - [x] `doc_url` is present
    https://holoviews.org/getting_started/index.html
11. - [x] Verify that the `build_number` is correct
12. - [x] has `setuptools`
13. - [x] has `wheel`
14. - [x] `pip` in test
15. - [x] Verify the test section
16. - [x] Verify if the package is `architecture specific`
17. - [x] Verify that private modules are not mentioned in the recipe For example: (_private_module)
18.  - [x] license_file: LICENSE.txt is present
19. - [x] License: BSD-3-Clause
    - [x] License is `spdx` compliant
 * Check if the license identifier has correct name from the SPDX License List

| Identifier                           |
|:-------------------------------------|
| BSD-3-Clause                         |
| BSD-3-Clause-Attribution             |
| BSD-3-Clause-Clear                   |
| BSD-3-Clause-LBNL                    |
| BSD-3-Clause-Modification            |
| BSD-3-Clause-No-Military-License     |
| BSD-3-Clause-No-Nuclear-License      |
| BSD-3-Clause-No-Nuclear-License-2014 |
| BSD-3-Clause-No-Nuclear-Warranty     |
| BSD-3-Clause-Open-MPI                |

20. - [x] license_family BSD is present
</details>

**Links:**
* [Anaconda Recipes feedstock](https://github.com/AnacondaRecipes/holoviews-feedstock)
* [Update branch](https://github.com/AnacondaRecipes/holoviews-feedstock/tree/1.15.2)
* [conda-forge recipe](https://github.com/{upstream}/{feedstock_name}-feedstock)
* [PyPI](https://pypi.org/project/holoviews)
* [Diff between upstream and feature branch](https://github.com/conda-forge/holoviews-feedstock/compare/main...AnacondaRecipes:1.15.2)
* [Diff between origin and feature branch](https://github.com/AnacondaRecipes/holoviews-feedstock/compare/master...AnacondaRecipes:1.15.2)
* [The last merged PRs](https://github.com/AnacondaRecipes/holoviews-feedstock/pulls?q=is%3Apr+is%3Amerged+sort%3Aupdated-desc+)

**Updating the recipe:**
If the recipe needs additional modification the update branch can be modified. Note that the PR diffs are not updated with these changes.
```
git clone -b 1.15.2 git@github.com:AnacondaRecipes/holoviews-feedstock.git
```

### CODE REVIEW

**Comment by jlstevens** [comment]
Looks good though are there no older param versions to worry about? Why get rid of the `param >=1.9.3` pin?

**Comment by skupr-anaconda** [comment]
> Looks good though are there no older param versions to worry about? Why get rid of the `param >=1.9.3` pin?

We try to not use pinnings in `host`. Then `conda-build` picks up param 1.12.0 for `host`.
If there will be the upper bound or exact version needed then they can be applied. 
Pinning for `param` is already in `run`

**Comment by jlstevens** [comment]
Ah didn't notice this was in the `host` section, thanks! Looks good!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,14 +21,15 @@ requirements:
   host:
     - python
     - pip
-    - pyct >=0.4.4
-    - param >=1.9.3
-    - setuptools >=30.3.0
+    - pyct
+    - param <2.0
```
Comment (katietz) [inline recipe/meta.yaml]: is this contrain required?

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: The conda-forge uses the upper bound. But I don't see that param project is developing v2.*.*
I'll remove it

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**